### PR TITLE
added a func to get an arbitrary child node

### PR DESCRIPTION
**Pull request by makotonakai**
- Added the new function so that a user can choose any child node he/she wants to choose 
(e.g. if I want to choose the child node in the middle of its list like the following)
```
・
・
<div class="narrowWrap" `data-sec="prg_dtl"></div>
<div class="narrowWrap"></div>
・
・
```
- Deleted the two existing functions (firstChild and lastChild) 
- Fixed README.md

### CODE REVIEW

**Comment by dsh0416** [comment]
Instead of breaking API changes, what about just adding a new API for `getChild(index)`. Otherwise, the existing users of this library may be confused.

**Comment by makotonakai** [comment]
@dsh0416 You're right.  I wasn't supposed to delete the existing methods (even though I fixed the README)

File: index.js [hunk 1]
```diff
@@ -335,7 +335,7 @@ $inherit(HTMLElement, Node, {
    * Get first child node
    * @return {Node} first child node
    */
-  get firstChild() {
+   get firstChild() {
```
Comment (dsh0416) [inline index.js:338]:
```suggestion
  get firstChild() {
```

File: index.js [hunk 2]
```diff
@@ -347,6 +347,15 @@ $inherit(HTMLElement, Node, {
     return this.childNodes.back;
   },
 
+  /**
+   * Get an arbitrary childNode
+   * @param  {int}  nodeIndex the index of the childNode you get
+   * @return {Node}       the childNode you get
+   */
+   getChild: function(nodeIndex) {
```
Comment (dsh0416) [inline index.js]:
```suggestion
  getChild: function(nodeIndex) {
```

File: README.md [hunk 3]
```diff
@@ -116,13 +116,15 @@ Query CSS Selector to find matching node.
 Append a child node to childNodes
 
 ### HTMLElement#firstChild
-
```
Comment (dsh0416) [inline README.md]: Removal of this line is unnecessary.

File: README.md [hunk 4]
```diff
@@ -116,13 +116,15 @@ Query CSS Selector to find matching node.
...
 Get first child node
 
 ### HTMLElement#lastChild
-
```
Comment (dsh0416) [inline README.md:123]: Removal of this line is unnecessary.

**Review by dsh0416 — COMMENTED** [review]
Some style changes.

File: index.js (hunk 1 above)
Comment (dsh0416) [reply index.js:338]: Keep indents with other function defs in the project.

File: index.js (hunk 2 above)
Comment (dsh0416) [reply index.js]: Keep indents with other function defs in the project.

File: index.js (hunk 1 above)
Comment (dsh0416) [reply index.js:338]: @MakotoNakai The later one is resolved, but not this one. It still has an extra space before `get`

**Comment by ashi009** [comment]
Thanks for the PR. However, I don't see why this method is needed, as the node is already accessible through the `childNodes`.

**Comment by makotonakai** [comment]
Thank you for your reply.  
Actually I didn't realize that until you pointed that out, but when I looked at the 'README.md', I thought that I could only get either the first or the last childNode, which was confusing at least for me.   
So, if I turn it into a new method and put that on the 'README.md', people will spend less time to solve the problem that I faced.

**Comment by ashi009** [comment]
The current API design follows the [DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model) API design, and presumably already familiar to most users. Therefore, I'll close this PR. Thanks again for writing it up.

**Pull request closed without merging**